inst/private/doctest_run_tests.m

```diff
@@ -17,25 +17,37 @@
 % Implementation note: all variables should start with
 % "DOCTEST__" as these will be available to the tests.
 
+  % Call subfcns at least once, workaround for test doing early "clear all"
```

Comment:
Is this because `clear all` clears the subfunctions from memory if they have not been `mlock`ed previously (per your `mlock` lines in the subfunctions)?

Reply:
It seems that when we do a single `mlock` here then we can remove all other `mlocks`s from the subfunctions.
Will test some more and then commit this change to the branch.